### PR TITLE
GAWB-1538 passthrough to rawls to cromwell to get cromwell version

### PR DESCRIPTION
**Pull request by helgridly**
rawls pr: https://github.com/broadinstitute/rawls/pull/641

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -2430,6 +2430,29 @@ paths:
         200:
           description: OK
 
+  /api/version/executionEngine:
```
Comment (MatthewBemis) [inline src/main/resources/swagger/api-docs.yaml]: paths should be organized alphabetically per https://github.com/broadinstitute/firecloud-orchestration/blob/develop/src/main/resources/swagger/README.md

File: src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala [hunk 2]
```diff
@@ -207,6 +207,11 @@ trait WorkspaceApiService extends HttpService with FireCloudRequestBuilding
             }
           }
         }
+      } ~
+      path("version" / "executionEngine") {
```
Comment (MatthewBemis) [inline src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala:211]: i don't think they're the most useful tests but maybe add a passthrough test for this to `WorkspaceApiServiceSpec.scala`?

**Review by MatthewBemis — CHANGES_REQUESTED** [review]

File: src/main/resources/swagger/api-docs.yaml (hunk 1 above)
Comment (helgridly) [reply src/main/resources/swagger/api-docs.yaml]: Woah. I had no idea this file was here!~

**Comment by helgridly** [comment]
@MatthewBemis donezo

**Review by MatthewBemis — APPROVED** [review]

**Pull request merged**